src/components/myCohortExercises/index.js

```diff
@@ -1,21 +1,24 @@
 import Card from "../card";
 import Button from "../button";
 import "./style.css"
+import { useTranslation } from "react-i18next";
 
 const MyExercises = () => {
+    const {t} = useTranslation()
+
     return (
         <>
             <Card>
                 <h3 className="my-cohort-exercises--header">My Exercises</h3>
                 <div>
                     <div className="my-cohort-exercises--stats">
-                        <p>Modules:</p> <span>2/7 completed</span>
-                        <p>Units:</p> <span>4/10 completed</span>
-                        <p>Exercises:</p> <span>34/58 completed</span>
+                        <p>{t("modules")}:</p> <span>2/7 {t("completed")}</span>
+                        <p>{t("units")}:</p> <span>4/10 {t("completed")}</span>
+                        <p>{t("exercises")}:</p> <span>34/58 {t("completed")}</span>
```

Comment:
There may be some languages which go like "completed 2/7". We'll need to account for this.